src/EventStore.Core/GossipStatsCollector.cs

```diff
@@ -0,0 +1,110 @@
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace EventStore.Core;
+
+public readonly struct GossipStats {
```

Comment:
You can simplify this by using a record

```
public record struct GossipStats(string Name, double MovingAverage, double Max, double Min, double Last);
```

Reply:
Thanks. Turned out I no longer need that record anymore